src/main/resources/Resources.properties

```diff
@@ -70,3 +70,8 @@ rhoLabelText = \u03c1
 wetherillTypeLabel      = Wetherill
 teraWasserburgTypeLabel = Tera-Waserburg
 noneTypeLabel           = None
+
+# Instructions Panel
+importContent = To start, go to 'File' > 'Import from TSV', and select the .tsv file you want to load.
+createTab = Alternatively, you can click on 'Create a Data Table', copy the data directly from your .tsv file and then click on 'Paste from Clipboard'.
+drawChart = Once you've imported your data, you can go to 'Charts' menu and select a chart.
```

Comment:
"Long text like this is better in a `.properties` file. Make `InstructionsPanel.properties` in this same package."